test/src/sender/main.cpp

```diff
@@ -105,13 +105,13 @@ cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp_data),typeid(outp_pa
 cadmium::dynamic::modeling::Models submodels_TOP = {generator_con, generator_ack, sender1};
 cadmium::dynamic::modeling::EICs eics_TOP = {};
 cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<Sender_defs::packetSentOut,outp_pack>("sender1"),
-cadmium::dynamic::translate::make_EOC<Sender_defs::ackReceivedOut,outp_ack>("sender1"),
-cadmium::dynamic::translate::make_EOC<Sender_defs::dataOut,outp_data>("sender1")
+  cadmium::dynamic::translate::make_EOC<sender_defs::packetSentOut,outp_pack>("sender1"),
+cadmium::dynamic::translate::make_EOC<sender_defs::ackReceivedOut,outp_ack>("sender1"),
+cadmium::dynamic::translate::make_EOC<sender_defs::dataOut,outp_data>("sender1")
 };
 cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Sender_defs::controlIn>("generator_con","sender1"),
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Sender_defs::ackIn>("generator_ack","sender1")
+  cadmium::dynamic::translate::make_IC<iestream_input_defs<message_t>::out,sender_defs::controlIn>("generator_con","sender1"),
```

Comment:
Many lines in this file are more than 80 characters in length (like this line)

Reply:
Changes done. Thank You.